chapters/tags-and-releases.qmd

```diff
@@ -221,6 +221,7 @@ To upload a Git repository to Zenodo, you first create a Zenodo account.
 On the Zenodo [start page](https://zenodo.org), you can click on "**Sign Up**" in the top right corner.
 Here you either can directly use your **GitHub account** to sign up, or sign up using **your email** (or other accounts) and link your GitHub account later on.
 To **link your GitHub account**, you can click on the arrow, next to your email, on the start page and select "GitHub".
+If you create an account on the normal [Zenodo](https://zenodo.org) site, you will have to create a new one to use the [Zenodo Sandbox](https://sandbox.zenodo.org), since they do not transfer.
```

Comment:
```suggestion
If you create an account on the [regular Zenodo site](https://zenodo.org), you will have to create a new account to use the [Zenodo Sandbox](https://sandbox.zenodo.org), since the accounts do not synchronize.
```